tests/test_storage.py

```diff
@@ -16,6 +16,9 @@
 
 
 def test_postgres():
+    if os.getenv("OPENAI_API_KEY") is None:
+        return
+
     config = MemGPTConfig()
```

Comment:
@sarahwooders how can we modify this to use local embeddings?

then we can add a new test, "test_postgres_local", and only the "test_postgres(_openai))" does a soft fail